### PR TITLE
Participants script no longer needs an API key. Update participants.

### PR DESCRIPTION
**Pull request by gkellogg**
[GitHack version](https://raw.githack.com/w3c/rdf-canon/update-participants/spec/index.html#informative-references).

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-canon/pull/151.html" title="Last updated on Jul 28, 2023, 2:01 AM UTC (ab682d8)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-canon/151/a7402d1...ab682d8.html" title="Last updated on Jul 28, 2023, 2:01 AM UTC (ab682d8)">Diff</a>

### CODE REVIEW

**Review by dlongley — APPROVED** [review]

File: spec/common/participants.html
Comment (seabass-labrax) [inline spec/common/participants.html:1]: Is it intentional for Andy Seaborne not to appear after this change?

Comment (dlongley) [reply spec/common/participants.html:1]: Hmm, I missed that. I do see he's not a [current participant](https://www.w3.org/groups/wg/rch/participants/) but was a [former one](https://www.w3.org/groups/wg/rch/former-participants/). The text implies that inclusion of current and former, it seems, which means he should be on the list, right?

Comment (gkellogg) [reply spec/common/participants.html:1]: @afs left the WG, and doesn’t get downloaded by the script anymore. We can and should add a specific acknowledgement for Andy’s contributions.

File: spec/common/participants.html [hunk 1]
```diff
@@ -1 +1 @@
-Members of the RDF Dataset Canonicalization and Hash Working Group Group included Ahamed Azeem, Ahmad Alobaid, Andy Seaborne, Benjamin Goering, Brent Zundel, Dan Brickley, Dan Yamamoto, Dave Longley, David Lehn, Gregg Kellogg, Ivan Herman, Jesse Wright, Kazue Sako, Leonard Rosenthol, Mahmoud Alkhraishi, Manu Sporny, Markus Sabadello, Michael Prorock, Phil Archer, Pierre-Antoine Champin, Sebastian Crane, Ted Thibodeau, Timothée HAUDEBOURG, and Tobias Kuhn.
+Members of the RDF Dataset Canonicalization and Hash Working Group Group included Ahamed Azeem, Ahmad Alobaid, Benjamin Goering, Brent Zundel, Dan Brickley, Dan Yamamoto, Dave Longley, David Lehn, Gregg Kellogg, Ivan Herman, Jesse Wright, Kazue Sako, Leonard Rosenthol, Mahmoud Alkhraishi, Manu Sporny, Markus Sabadello, Michael Prorock, Phil Archer, Pierre-Antoine Champin, Sebastian Crane, Ted Thibodeau, Timothée Haudebourg, and Tobias Kuhn.
```
Comment (gkellogg) [inline spec/common/participants.html:1]: We could update the script to generate "At the time of publication, members of the ...". Or, if there is an API endpoint that includes both current and former members, we could use that.

Comment (yamdan) [reply spec/common/participants.html:1]: It seems that we can use `former` flag to include both current and former members:
https://api.w3.org/groups/wg/rch/users?former=true

**Review by iherman — CHANGES_REQUESTED** [review]
I think that Aidan Hogan should be listed in the first paragraph of the acknowledgement section, alongside Jeremy, TimBL, or Fisteus. After all, his work is the closest to RDFC. (I must admit I did not even know about Fisteus' work).

File: spec/index.html [hunk 2]
```diff
@@ -3632,12 +3632,15 @@ <h2>Changes since the First Public Working Draft of 24 November 2022</h2>
...
-
+    graph canonicalization problem,
+    Andy Seaborne and Gavin Carothers for providing valuable feedback and testing input
+    for the algorithm defined in this specification,
+    Sir Tim Berners-Lee for his thoughts on graph canonicalization over the years, and
+    Jesús Arias Fisteus for his work on a similar algorithm.</p>
+
+  <p>Aiden Hogan's [[Hogan-Canonical-RDF]] provided an important contemporary
+    analysis of the canonicalization problem provided an independent
+    justification of the development of <a>RDFC-1.0</a>.</p>
```
Comment (gkellogg) [inline spec/index.html:3643]: @iherman thanks for mentioning this omission. There is an open issue on referencing this as part of the history of development, but a specific reference to the work had not yet been added..

File: spec/common/participants.html (hunk 1 above)
Comment (gkellogg) [reply spec/common/participants.html]: @yamdan thanks for the pointer; I added it to the script and regenerated the participants.

File: spec/index.html (hunk 2 above)
Comment (TallTed) [reply spec/index.html:3643]: This paragraph appears to be an amalgam of two sentences (note the two occurrences of `provided`). I don't know enough about the particulars to provide an edit.

**Review by seabass-labrax — CHANGES_REQUESTED** [review]

File: spec/index.html (hunk 2 above)
Comment (seabass-labrax) [reply spec/index.html:3643]: Addressed; please see other comment thread.

**Review by yamdan — COMMENTED** [review]
Thanks @gkellogg.
This might be apart from this PR, but it seems a bit strange for me that the resumed reference of `[Hogan-Canonical-RDF]` in G.2 Informative references contains the word "unofficial", because Hogan's paper was published officially in ACM.
I found it comes from the `status` property in `spec/common/biblio.js`, whose I do not know well so forgive me if this is my misunderstanding.

**Comment by gkellogg** [comment]
Probably need to look for other examples in specref. It’s “unofficial” in the standards sense, but there may be another appropriate status to use. I don’t think we’re bound to any specific usage requirements.

**Comment by gkellogg** [comment]
> I found it comes from the `status` property in `spec/common/biblio.js`, whose I do not know well so forgive me if this is my misunderstanding.

Currently, the "status" field is "unofficial", and the "publisher" is "ACM Transactions on the Web". It could instead be "status": "Transactions on the Web, Volume 11, Issue 4", "publisher": "ACM".

@iherman do you have a better feel for this?

**Comment by iherman** [comment]
> "Transactions on the Web, Volume 11, Issue 4", "publisher": "ACM"

This looks o.k. to me, except that I believe the year should also be added.

**Comment by TallTed** [comment]
Details should be updated based on https://dl.acm.org/doi/10.1145/3068333. They provide citations for copy-and-paste, as below:

### ACM Ref
`Aidan Hogan. 2017. Canonical Forms for Isomorphic and Equivalent RDF Graphs: Algorithms for Leaning and Labelling Blank Nodes. ACM Trans. Web 11, 4, Article 22 (November 2017), 62 pages. https://doi.org/10.1145/3068333`
### BibTeX
```
@article{10.1145/3068333,
author = {Hogan, Aidan},
title = {Canonical Forms for Isomorphic and Equivalent RDF Graphs: Algorithms for Leaning and Labelling Blank Nodes},
year = {2017},
issue_date = {November 2017},
publisher = {Association for Computing Machinery},
address = {New York, NY, USA},
volume = {11},
number = {4},
issn = {1559-1131},
url = {https://doi.org/10.1145/3068333},
doi = {10.1145/3068333},
abstract = {Existential blank nodes greatly complicate a number of fundamental operations on Resource Description Framework (RDF) graphs. In particular, the problems of determining if two RDF graphs have the same structure modulo blank node labels (i.e., if they are isomorphic), or determining if two RDF graphs have the same meaning under simple semantics (i.e., if they are simple-equivalent), have no known polynomial-time algorithms. In this article, we propose methods that can produce two canonical forms of an RDF graph. The first canonical form preserves isomorphism such that any two isomorphic RDF graphs will produce the same canonical form; this iso-canonical form is produced by modifying the well-known canonical labelling algorithm Nauty for application to RDF graphs. The second canonical form additionally preserves simple-equivalence such that any two simple-equivalent RDF graphs will produce the same canonical form; this equi-canonical form is produced by, in a preliminary step, leaning the RDF graph, and then computing the iso-canonical form. These algorithms have a number of practical applications, such as for identifying isomorphic or equivalent RDF graphs in a large collection without requiring pairwise comparison, for computing checksums or signing RDF graphs, for applying consistent Skolemisation schemes where blank nodes are mapped in a canonical manner to Internationalised Resource Identifiers (IRIs), and so forth. Likewise a variety of algorithms can be simplified by presupposing RDF graphs in one of these canonical forms. Both algorithms require exponential steps in the worst case; in our evaluation we demonstrate that there indeed exist difficult synthetic cases, but we also provide results over 9.9 million RDF graphs that suggest such cases occur infrequently in the real world, and that both canonical forms can be efficiently computed in all but a handful of such cases.},
journal = {ACM Trans. Web},
month = {jul},
articleno = {22},
numpages = {62},
keywords = {skolemisation, signing, linked data, isomorphism, Semantic web}
}
```
### EndNote
```
%0 Journal Article
%T Canonical Forms for Isomorphic and Equivalent RDF Graphs: Algorithms for Leaning and Labelling Blank Nodes
%V 11
%@ 1559-1131
%U https://doi.org/10.1145/3068333
%R 10.1145/3068333
%N 4
%J ACM Trans. Web
%I Association for Computing Machinery
%A Aidan Hogan
%D 2017
%P Article 22
%K linked data, isomorphism, skolemisation, Semantic web, signing
```

**Comment by gkellogg** [comment]
I added a GitHack link in the opening to properly see the rendered bibliographic reference.

**Review by TallTed — APPROVED** [review]
I do wonder what the policy is on linking to the paper, i.e., whether we should link to Hogan's personal pre-publication PDF, or to ACM's (paywalled) site.

**Comment by gkellogg** [comment]
IIRC, the ACM link hits a paywall, and this link goes straight to Aiden’s public ally available version. I think that’s been done in the past for other paywall links, where there’s an open version available.

File: spec/common/biblio.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ const rch = {
       id: "DesignIssues-Diff",
       title: "Delta: an ontology for the distribution of differences between RDF graphs",
       authors: ["Tim Berners-Leee"],
-      date: "2015-09-25",
+      date: "Septmber 25, 2015",
```
Comment (yamdan) [inline spec/common/biblio.js:22]:
```suggestion
      date: "September 25, 2015",
```

**Review by yamdan — APPROVED** [review]
Thanks! added a small typo fixing.

**Pull request merged**